wire-compiler/src/main/java/com/squareup/wire/KotlinFileWriter.kt

```diff
@@ -57,10 +52,45 @@ internal class KotlinFileWriter(
       try {
         kotlinFile.writeTo(path)
       } catch (e: IOException) {
+        val className = when (next) {
+          is Type -> kotlinGenerator.generatedTypeName(next).canonicalName
+          is Service -> next.type().toString()
+          else -> next.toString()
+        }
         throw IOException(
-            "Error emitting ${kotlinFile.packageName}.${className.canonicalName} to $destination",
+            "Error emitting ${kotlinFile.packageName}.$className to $destination",
             e)
       }
+      }
     }
+
+  private fun generateFileForType(type: Type): FileSpec {
+    val typeSpec = kotlinGenerator.generateType(type)
+    val className = kotlinGenerator.generatedTypeName(type)
+    return FileSpec.builder(className.packageName, typeSpec.name!!)
+        .addComment(WireCompiler.CODE_GENERATED_BY_WIRE)
```

Comment:
`.setIndent("  ")`!!!!!!!

Reply:
will do that for service now, and will follow up for types because noize